src/resmom/linux/alps.c

```diff
@@ -4530,8 +5274,19 @@ alps_create_reserve_request(job *pjob, basil_request_reserve_t **req)
 
 		p->depth = ns->depth;
 		p->nppn = width = ns->width;
```

Comment:
It's not required, as per basil 1.3 documentation
"If the nppcu value is unspeciﬁed, a default of zero will be assumed. Since pre-BASIL 1.3 implementors are unaware of CUs and cannot specify the nppcu value, a default of zero is also used in this environment."